compiler/lib/src/main/scala/codegen/CppWriter/ComponentCppWriter/ComponentCppWriterUtils.scala

```diff
@@ -445,6 +475,15 @@ abstract class ComponentCppWriterUtils(
       )
       case _ => portParamTypeMap(p.getUnqualifiedName).map((n, tn, t) => (n, tn, Some(t)))
     }
+  
+  /** Get port params as a list of names */
+  def getPortParamNames(p: PortInstance): List[String] =
+    p.getType match {
+      case Some(PortInstance.Type.Serial) => List(
+        "buffer"
```

Comment:
Nit: see if magic string can be re-used with other places

Reply:
Will not fix, larger implications on code base.